ConfirmationDialog unit test error on Windows.

TRACEBACK:

```
GUI closing
Traceback (most recent call last):
  File "D:\jenkins\workspace\Test_tribler_devel_win2008r2@2\tribler\Tribler\Main\Dialogs\ConfirmationDialog.py", line 44, in OnCancel
    self.EndModal(wx.ID_CANCEL)
  File "C:\Python27\x86\lib\site-packages\wx-2.8-msw-unicode\wx\_windows.py", line 715, in EndModal
    return _windows_.Dialog_EndModal(*args, **kwargs)
wx._core.PyAssertionError: C++ assertion "IsModal()" failed at ..\..\src\msw\dialog.cpp(361) in wxDialog::EndModal(): EndModal() called for non modal dialog
GUI closed
```

--- a/Tribler/Main/Dialogs/ConfirmationDialog.py
+++ b/Tribler/Main/Dialogs/ConfirmationDialog.py
@@ -33,8 +33,6 @@ class ConfirmationDialog(wx.Dialog):
         mainSizer.Add(buttonSizer, 0, wx.EXPAND | wx.ALL, 15)
         self.SetSizer(mainSizer)
 
-        self.Bind(wx.EVT_CLOSE, self.OnCancel)
-
     def OnConfirm(self, event):
         if self.checkbox.GetValue():
             GUIUtility.getInstance().WriteGuiSetting('show_%s' % self.GetName(), False)

